main/MamdaniController.cpp

```diff
@@ -0,0 +1,121 @@
+/**
+ * @file MamdaniController.cpp
+ * @author ACMAX (aavaloscorrales@gmail.com)
+ * @brief 
+ * @version 0.1
+ * @date 2024-10-29
+ * 
+ * @copyright Copyright (c) 2024
+ * 
+ */
+
+#ifndef MAMDANI_CONTROLLER_CPP
+#define MAMDANI_CONTROLLER_CPP
+
+#include "MamdaniController.hpp"
+
+float MamdaniController::operator() (float error) {
```

Comment:
Missing comments explaing operator overload and use cases

Reply:
The comment explaining the operator is in the header file.